system-parachains/people-paseo/src/people.rs

```diff
@@ -242,3 +250,148 @@ impl Default for IdentityInfo {
 		}
 	}
 }
+
+parameter_types! {
+	// TODO: good values and reasoning
+	pub const OriginRestrictionAllowanceMax: Balance = 100 * UNITS;
+	pub const OriginRestrictionAllowanceRecovery: Balance = UNITS / 100;
```

Comment:
Let's do 1 UNIT every minute, so this would be UNITS / 10 with 10 blocks every minute.

Reply:
yeah I blindlty copy-pasted, I think 1 UNIT per minute seems way too much though.

For 50,000 users it is 72,000,000 UNITS per day. This should completely saturate the network. if the current fee are low I would do 1 UNIT per day per person maybe.

The recovery rate should correspond to the average usage for one person we want to have free, the max allowance should correspond to the peak usage for one person.

If we let user spend more than 10 UNIT per day on fees I think it is not sustainable. I will put 10 UNIT per day for now.